web_calendar_custom/static/src/js/web_calendar_custom.js

```diff
@@ -0,0 +1,141 @@
+/*
+    © 2018 Savoir-faire Linux <https://savoirfairelinux.com>
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL.html).
+*/
+
+
+odoo.define('web_calendar_custom.renderer_setting', function (require) {
+    "use strict";
+
+    var rpc = require('web.rpc');
+    var AbstractRenderer = require('web.AbstractRenderer');
+    var core = require('web.core');
+
+    var _t = core._t;
+
+    /* eslint no-extend-native: "error" */
+    String.prototype.toHHMMSS = function () {
+        var hour_num = parseFloat(this);
+        var hours = Math.trunc(hour_num);
+        var minutes = (hour_num - hours) * 60;
+
+        if (hours < 10) {
+            hours = '0' + hours;
+        }
+        if (minutes < 10) {
+            minutes = '0' + minutes;
+        }
+        return hours + ':' + minutes + ':' + '00';
+    }
+
+    /* Get configuration values from res.config.settings */
+    rpc.query({
```

Comment:
Put this in `willStart` of `AbstractRenderer`

Reply:
Hi @tarteo I tried to put the rpc call in willStart and put the rest in Start. Also and changed AbstractRenderer.include to AbstractRenderer.extend but the code won't execute. How can I contact you for some advise?
Regards.

Reply:
I finally added all the code in willStart and Start function by using include AbstractRenderer to extend the functions used for applying the new settings to the calendar. And it worked. However I still have to use a global variable to pass rpc call results, from willStart function to Start function. Regards, Larbi